.circleci/config.yml

```diff
@@ -65,6 +65,35 @@ jobs:
             yarn run spec
             git diff --exit-code --name-only ./api-specifications/properties.json
 
+  api-governance:
+    machine: true
+    working_directory: ~/project
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/project
+      - run:
+          name: Prepare API Compliance
+          command: |
+            docker pull ghcr.io/qlik-download/api-compliance
+            docker create -v /specs --name specs alpine:3.4 /bin/true
+            docker cp ./api-specifications/properties.json specs:/specs
+
+      - run:
+          name: Run API Compliance
+          command: >
+            VER=v$(cat ../package.json | jq -r '.version')
```

Comment:
Should it not be `./package.json`?

Reply:
You were right @cbt1 , that was making the version to be empty. Now it's fixed :)